main.ledger

```diff
@@ -1301,6 +1301,11 @@
     Liabilities:Reimbursement:Max Wofford
     ; Receipt: d019741e762b90c4a83235e074ea0c71.jpg
 
+2015/09/14 "Mission Reds LLC"
```

Comment:
Ah, our fiscal sponsor accidentally moved money into our account. This should fix that.